osu.Game.Tests/Visual/Components/TestSceneIdleTracker.cs

```diff
@@ -21,38 +22,41 @@ public class TestSceneIdleTracker : OsuManualInputManagerTestScene
 
         private IdleTrackingBox[] boxes;
 
+        private SessionStatics sessionStatics;
+
         [SetUp]
         public void SetUp() => Schedule(() =>
         {
+            sessionStatics = new SessionStatics();
             InputManager.MoveMouseTo(Vector2.Zero);
 
             Children = boxes = new[]
             {
-                box1 = new IdleTrackingBox(2000)
+                box1 = new IdleTrackingBox(2000, sessionStatics)
                 {
                     Name = "TopLeft",
                     RelativeSizeAxes = Axes.Both,
                     Colour = Color4.Red,
                     Anchor = Anchor.TopLeft,
                     Origin = Anchor.TopLeft,
                 },
-                box2 = new IdleTrackingBox(4000)
+                box2 = new IdleTrackingBox(4000, sessionStatics)
                 {
                     Name = "TopRight",
                     RelativeSizeAxes = Axes.Both,
                     Colour = Color4.Green,
                     Anchor = Anchor.TopRight,
                     Origin = Anchor.TopRight,
                 },
-                box3 = new IdleTrackingBox(6000)
+                box3 = new IdleTrackingBox(6000, sessionStatics)
                 {
                     Name = "BottomLeft",
                     RelativeSizeAxes = Axes.Both,
                     Colour = Color4.Blue,
                     Anchor = Anchor.BottomLeft,
                     Origin = Anchor.BottomLeft,
                 },
-                box4 = new IdleTrackingBox(8000)
+                box4 = new IdleTrackingBox(8000, sessionStatics)
```

Comment:
i don't get what's going on here.

you probably want to revert changes on this test scene and make a new headless test which is testing what you want, without this box stuff (ie. why is the `sessionStatics` being reset by four different drawable instances here?)

Reply:
I initially did this because it initially modified the IdleTracker itself and I didnt know that there were non-visual tests I could add to, but now I guess it *does* makes better sense to make a new test entirely.